test/mocks/deps/one.js

```diff
@@ -14,3 +14,16 @@ let a = require('@a/package')
 let b = require('@b/package/method')
 let c = require('c/specific/method')
 let d = require('d')
+
+// Minifiers may redefine variables to save space
+let missingInit
+missingInit = 'ignoredmodule'
+require(missingInit)
```

Comment:
In general I hope we'll do our best to warn folks; in this change I don't think we've successfully warned the user about these missing requires, have we?

Reply:
I should verify that!